CHANGELOG.md

```diff
@@ -105,7 +105,40 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Metroid Prime
 
-- Changed: The map tracker uses the same names for elevators as when editing a preset.
+- Fixed: Crashes in several rooms when pressing start to skip cutscene exactly 1 second from the end of cutscene
+- Fixed: Incorrect shield texture for vertical **Power Beam Only** doors
+- Fixed: Elite Research - Stutter when loading room
+- Fixed: Ruined Shrine Access - Still playing Beetle fight music after leaving
+- Fixed: Mine Security Station - Wave Pirates not dropping down if the player didn't skip the cutscene immediately
+- Fixed: Uncrashed Frigate - Escape music sometimes playing at inappropriate times
+- Fixed: Furnace - Ghost elements re-appearing when re-entering from East Furnace Access
+- Fixed: Main Plaza - Vault Ledge door cosmetic fix
+- Fixed: Hive Totem - Incorrect water height when *not* skipping cutscene, improved cutscene skip timing
```

Comment:
116
Inaccurate, would be better to just leave it at "Improved intro Cutscene Skip" and maybe change it from Fix to Change.